src/entities/activity/ui/items/ActionPlan/ResponseSegment.tsx

```diff
@@ -24,7 +24,7 @@ const isAnswersSkipped = (answers: string[]): boolean => {
 type FieldValueTransformer = (value: string) => string;
 const identity: FieldValueTransformer = (value) => value;
 
-type FieldValuesJoiner = (values: string[]) => string;
+type FieldValuesJoiner = (values: string[]) => string | JSX.Element[];
```

Comment:
Nit: I often use the `ReactNode` type to represent "any rendered JSX content", it's a very flexible and convenient type to use:
```suggestion
type FieldValuesJoiner = (values: string[]) => ReactNode;
```